.env.example

```diff
@@ -23,5 +23,8 @@ Env="pre" #the mocked environment will mock out your https calls as defined in m
 
 RedisServer="redis://localhost:6379"
 RedisCacheTimeSeconds="600"
+RedisPort="6379"
+
 ResourcePrefix="http://localhost:5173" #the url that serves the widget ui
 ResourceVersion=""
+ResourcePort="5173"
```

Comment:
I added an env specifically for the port, because the docker compose file uses different conventions for specifying the host (not localhost).